day_10/task_1/tests/pipe_maze_tests.cpp

```diff
@@ -5,5 +5,16 @@ TEST(PipeMazeToolsShould, FindTheStartingPoint)
 {
     const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
 
-    ASSERT_EQ(find_starting_point(pipe_maze), (std::make_pair<int, int>(1, 1)));
+    ASSERT_EQ(find_starting_point(pipe_maze), MazePoint(1, 1));
+}
+
+TEST(PipeMazeToolsShould, FindNextNeighbors)
+{
+    const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
```

Comment:
`pipe_maze` is used in the previous test also. We could avoid DRY violation by creating test fixture with `pipe_maze`. Could you apply such fixture, please?

Reply:
done